Rider/app/build.gradle

```diff
@@ -7,7 +7,7 @@ apply plugin: 'kotlin-android-extensions'
 android {
     compileSdkVersion 28
     defaultConfig {
-        applicationId "com.example.rider"
+        applicationId "com.dti.rider"
         minSdkVersion 15
```

Comment:
This is way too low and means lots of functionalities will be unavailable; at least have Lollipop